dev/release/VERIFY.md

```diff
@@ -50,15 +50,28 @@ You can install them by the followings on Debian GNU/Linux and Ubuntu:
 You can install them by the followings on CentOS 7:
 
 ```console
-% sudo yum install -y gobject-introspection-devel
+% sudo yum install -y gobject-introspection-devel git curl
+% cd /etc/pki/rpm-gpg
+% curl -O http://springdale.princeton.edu/data/springdale/7/x86_64/os/RPM-GPG-KEY-springdale
+% cd /etc/yum.repos.d
+% cat << EOF > Springdale-SCL.repo
+[Springdale-SCL]
+name=Springdale - SCL
+baseurl=http://springdale.princeton.edu/data/springdale/SCL/$releasever/$basearch
+gpgcheck=1
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-springdale
+EOF
```

Comment:
Why do we need this?

Reply:
This is needed to get newer llvm versions.  It is faster than compiling from source.

Reply:
How about using EPEL for LLVM? It seems that EPEL provides LLVM 13.

(But I want to disable Gandiva when we verify a RC on CentOS 7...)

Reply:
EPEL for LLVM seems to be ok.

Reply:
Then, can we remove this repository?

Reply:
Yes. It has been removed.

Reply:
It seems that this still exists in this branch. Did you forget to push a commit?